src/main/java/com/epam/training/array/ArraySorter.java

```diff
@@ -1,8 +1,15 @@
 package com.epam.training.array;
 
+import java.util.Arrays;
+
 public class ArraySorter {
 
     public Array sort(Array array) {
-        throw new UnsupportedOperationException();
+        if (array.getItems() == null) {
+            throw new UnsupportedOperationException();
```

Comment:
own exception, UnsupportedOperationException() if your class doesn't support this method